### PR TITLE
Translate documentation to Spanish 

### LINKED ISSUE #3208: Spanish Translation needed

### PR DESCRIPTION
**Pull request by Akshay-Nagle**
FIXES #3208 Spanish Translation needed

### CODE REVIEW

**Comment by kichloo** [comment]
Hi, how are you doing the DCO checks? I mentioned it in the commits but it is not getting signed.

**Comment by Akshay-Nagle** [comment]
@prateekpandey14 Please review this PR

File: translations/README.es.txt [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+[! [Liberaciones] (https://img.shields.io/github/release/openebs/openebs/all.svg?style=flat-square)] (https://github.com/openebs/openebs/releases)
+[! [#openebs canal de Slack] (https://img.shields.io/badge/slack-openebs-brightgreen.svg?logo=slack)] (https://kubernetes.slack.com/messages/openebs)
+[! [Twitter] (https://img.shields.io/twitter/follow/openebs.svg?style=social&label=Follow)] (https://twitter.com/intent/follow?screen_name=openebs)
+[! [Bienvenidos a los PRP] (https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)] (https://github.com/openebs/openebs/blob/master/CONTRIBUTING.md)
+[! [Estado FOSSA] (https://app.fossa.com/api/projects/git%2Bgithub.com%2Fopenebs%2Fopenebs.svg?type=shield)] (https://app.fossa.com/projects/git%2Bgithub.com%2Fopenebs%2Fopenebs?ref=badge_shield)
+[! [Prácticas recomendadas de CII] (https://bestpractices.coreinfrastructure.org/projects/1754/badge)] (https://bestpractices.coreinfrastructure.org/projects/1754)
+
+https://openebs.io/
+
```
Comment (kmova) [inline translations/README.es.txt:11]: Please check that there is a change in how the other translations are listed.

Comment (Akshay-Nagle) [reply translations/README.es.txt]: @kmova I have made the changes that you asked. Please let me know if later commits make any problem.

**Comment by Akshay-Nagle** [comment]
@kmova if later commits, create a problem then allow me to create new PR and close this PR.

**Comment by Akshay-Nagle** [comment]
@kmova @prateekpandey14 I made a new PR #3271 which have zero conflicts. Please review it and add relevant tags.

**Pull request closed without merging**